amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "Describes a SLSA buildType, as defined in https://slsa.dev/provenance/v0.2.",
+  "type": "object",
+  "required": [
+    "buildType",
+    "buildConfig",
+    "materials"
+  ],
+  "not": {
+    "required": [
+      "invocation"
+    ]
+  },
+  "properties": {
+    "buildType": {
+      "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype.json"
+    },
+    "buildConfig": {
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "Command passed to the builder image.",
+          "examples": [
+            "./scripts/runner build-functions-server"
+          ]
+        },
+        "outputPath": {
+          "type": "string",
+          "description": "The path, relative to the root of the git repository, where the artifact built by the command is expected to be found.",
+          "examples": [
+            "./oak_functions/loader/bin/oak_functions_loader"
+          ]
+        },
+        "expectedArtifactDigest": {
+          "type": "object",
```

Comment:
Is it possible to clarify that it has to be DigestSet in https://github.com/in-toto/attestation/blob/main/spec/field_types.md#DigestSet? 

Do we have to define that object somewhere ourselves to be able to use json/toml validation?

Reply:
> Is it possible to clarify that it has to be DigestSet in https://github.com/in-toto/attestation/blob/main/spec/field_types.md#DigestSet?
Possible yes. Note that at the moment however this is a subset of the DigestSet, as it only accepts one type of hash. DigestSet allows arbitrary hashing functions. I'll add a comment to clarify. It is limited to `sha256` to allow our tooling to validate hashes. We can add more hash types in later versions. :) 

> Do we have to define that object somewhere ourselves to be able to use json/toml validation?
Yes